src/lib/components/FormControls/HelperText.tsx

```diff
@@ -1,7 +1,94 @@
 import classNames from 'classnames';
 import type { ComponentProps, FC, PropsWithChildren } from 'react';
-import type { FlowbiteColors } from '../Flowbite/FlowbiteTheme';
+import type { FlowbiteBoolean, FlowbiteColors } from '../Flowbite/FlowbiteTheme';
 import { useTheme } from '../Flowbite/ThemeContext';
+import { LabelColors } from './Label';
+import { SelectColors, SelectSizes } from './Select';
+import { TextareaColors } from './Textarea';
+import { TextInputColors, TextInputSizes } from './TextInput';
+
+export interface FlowbiteFormControlsTheme {
```

Comment:
Ideally, we will have to move the interfaces inside their own component folders.

Reply:
I created the respective tasks for every single component. We are going to handle it in separated PR.